docs/source/_includes/available_permission_types.rst

```diff
@@ -56,8 +56,6 @@ Global
 +----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
 | **execution_views_filters_list** | Ability view all the distinct execution filters.                                                                                                          |
 +----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
-| **key_value_pair_list**          | Ability to list (view all) Key-Value Pairs.                                                                                                               |
```

Comment:
These rbac changes are unrelated.
Are they changes in this file Auto generated?

Reply:
@cognifloyd  Good spot - it was picking up 3.5 st2 source rather than the 3.7dev - re-generated after cleaning up my cloned repo.